geteduroam/GeteduroamPackage/Sources/Models/LocalizedEntry.swift

```diff
@@ -1,17 +1,17 @@
 import Foundation
 
 public struct LocalizedEntry: Codable, Equatable, Sendable {
-    public init(language: String? = "any", value: String) {
+    public init(language: String? = "any", display: String) {
         self.language = language
-        self.value = value
+        self.display = display
     }
     
     let language: String?
-    let value: String
+    let display: String?
     
     enum CodingKeys: String, CodingKey {
-        case value = ""
         case language = "lang"
+        case display
```

Comment:
I think you could have avoided a lot of changes if you'd have done this instead: `case value = "display"`.